saleor/graphql/checkout/types.py

```diff
@@ -491,20 +491,27 @@ def apply_price_to_shipping_method(channel_listings):
     @traced_resolver
     def resolve_available_collection_points(root: models.Checkout, info):
         def get_available_collection_points(data):
-            address, lines, checkout_info = data
+            address, lines, checkout_info, channel = data
+            country_code = address.country.code if address else None
+
             return get_valid_collection_points_for_checkout(
-                lines, checkout_info, country_code=address.country.code
+                lines,
+                checkout_info,
```

Comment:
I guess this function doesn't need `checkout_info` anymore.

Reply:
That's true. Done